river/cluster/clustream_kernel.py

```diff
@@ -0,0 +1,151 @@
+import math
+from abc import ABCMeta
+
+EPSILON = 0.00005
+MIN_VARIANCE = 1e-50
+
+
+class ClustreamKernel(metaclass=ABCMeta):
+    def __init__(
+        self, x=None, sample_weight=None, cluster=None, timestamp=None, T=None, M=None
+    ):
+
+        self.T = T
```

Comment:
Please change to an explicit variable name to ease the maintenance of the code.

Reply:
I have changed `T` to `kernel_radius_factor` and `M` to `max_kernels`.